commands/help.py

```diff
@@ -0,0 +1,11 @@
+async def help(ctx):
+    """ Sends help message to the user, informing of available commands """
+    message = """
+Hey there! You need help?
+Here are the commands I respond to:
+** help ** - I will whisper you this help message.
+** roll ** - Nothing unique. I'll just roll a dice from 1-100. Good Luck!
+** weather [zip code] ** - Let you know the weather for zipcode, so you don't have to go outside.
+** nasa_apod ** - Displays the Astral Photo of the Day, provided by the NASA API.
```

Comment:
The description here is a bit off from the official documentation.
```suggestion
** nasa_apod ** - Displays the Astronomy Picture of the Day, provided by the NASA API.
```